src/current/_includes/releases/v22.2/v22.2.12.md

```diff
@@ -0,0 +1,22 @@
+## v22.2.12
+
+Release Date: July 24, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-12-sql-language-changes">SQL language changes</h3>
```

Comment:
nit: this would make more sense as a CLI change or general change. `debug doctor` isn't really a part of SQL.

Reply:
Done!